### DISCUSSION TITLE
Increase log level?

### DISCUSSION THREAD

**Discussion by miwurster** [plausible/analytics#3206] [Self-Hosted Support] [answered]
Hi, is there a way to increase the log level of the Plausible analytics instance?

Thanks in advance,
Michael

**Comment by ruslandoga** [comment, marked as answer]
👋 @miwurster 

Yes, you can use the `LOG_LEVEL` env var: https://github.com/plausible/analytics/blob/7c5ebab2c6e37d3b52ee8c6ad0d432cf189cc973/config/runtime.exs#L198-L201

It defaults to `warn` if not set and the available log levels are listed here: https://hexdocs.pm/logger/Logger.html#module-levels
